gcp_variant_transforms/libs/vcf_reserved_fields.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides reserved INFO and FORMAT fields based on VCF 4.3 spec.
+
+See http://samtools.github.io/hts-specs/VCFv4.3.pdf for more details.
+"""
+
+from __future__ import absolute_import
+
+from typing import Optional  # pylint: disable=unused-import
+
+from vcf import parser
+
+
+def _get_field_count(value):
+  # type: (str) -> Optional[int]
+  return parser.field_counts[value]
+
+
+def _create_format(field, number, field_type, description):
+   # type: (str, int, str, str) -> parser._Format
+  return parser._Format(field, number, field_type, description)
+
+
+def _create_info(field, number, field_type, description):
+  # type: (str, int, str, str) -> parser._Info
+  return parser._Info(field, number, field_type, description, None, None)
```

Comment:
nit: please add `id=field`, `num=number` etc so that you can specify what the two `None`s are for.

Reply:
Removed these functions.